runtime/browser/demo/explorer.html

```diff
@@ -0,0 +1,28 @@
+<link rel="import" href="../../../strategy-explorer/strategy-explorer.html"></link>
```

Comment:
Feel free to maneuver these files around as you like. I installed strategy-explorer in the CDN, so all the CDN based demos as so on will use it from there.